NorthstarDLL/core/hooks.cpp

```diff
@@ -392,8 +394,11 @@ HMODULE, WINAPI, (LPCSTR lpLibFileName, HANDLE hFile, DWORD dwFlags))
 {
 	HMODULE moduleAddress;
 
+	LPCSTR lpLibFileNameEnd = lpLibFileName + strlen(lpLibFileName);
+	LPCSTR lpLibName = lpLibFileNameEnd - strlen(XINPUT1_3_DLL);
```

Comment:
edge case:
if `lpLibFileName` is shorter than `XINPUT1_3_DLL` we run out of the buffer.

Reply:
fixed in ceccedd574b651d0eac5d82cbd357571b22f22ec